R/ForecastRegrTask.R

```diff
@@ -0,0 +1,57 @@
+#' @export
+#' @rdname Task
+#' @importFrom zoo index coredata
+makeForecastRegrTask = function(id = deparse(substitute(data)), data, target,
```

Comment:
No, just a sentence or two on what the task is. Ideally, we would have this for other tasks as well...